release-matrix/release-matrix-schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "MKE 4 Release Matrix",
+  "description": "",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "clusterVersion": {
+        "description": "The MKE 4 minor/patch version that this entry applies to",
+        "type": "string"
+      },
+      "blueprintOperatorVersion": {
+        "description": "Blueprint operator version for this MKE 4 release",
+        "type": "string"
+      },
+      "mkeOperatorVersion": {
+        "type": "string",
+        "description": "MKE Operator version for this MKE 4 release"
+      },
+      "k0sVersion": {
+        "type": "string",
+        "description": "k0s version for this MKE 4 release"
+      },
+      "isLatest": {
```

Comment:
@RaunakShah - why is this needed?

Reply:
@vikramhh as discussed in the orca cafe this week and also prior meetings, we want to allow users to specify "latest" as their upgrade version. This is to identify the latest supported release version.